tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -127,25 +136,43 @@ echo "Installing pip whl file: ${WHL_PATH}"
 
 # Create temporary directory for install test
 VENV_DIR="${PIP_TEST_ROOT}/venv"
-rm -rf "${VENV_DIR}" && mkdir -p "${VENV_DIR}"
-echo "Create directory for virtualenv: ${VENV_DIR}"
+if [[ -d "${VENV_DIR}" ]]; then
+  echo "virtualenv directory already exists: ${VENV_DIR}"
+else
+  echo "Creating virtualenv directory: ${VENV_DIR}"
+  mkdir -p ${VENV_DIR} || \
```

Comment:
You can do this, but is not overly verbose? I mean "mkdir -p ..." does not fail even if the directory is there already. Then all the if statements and echo statements are just noise.